helm/cluster-api-bootstrap-provider-kubeadm/templates/webhook/clusterrolebinding.yaml

```diff
@@ -1,4 +1,3 @@
-{{ if eq (include "resource.app.unique" .) "true" }}
```

Comment:
The rest is removing my broken attempt to fix in https://github.com/giantswarm/cluster-api-bootstrap-provider-kubeadm-app/pull/55 😬

Reply:
how does it work without the conditional? what prevents Helm from creating these manifests both for the release and the app collection?